health/service/src/main/java/com/cerner/beadledom/health/HealthDependency.java

```diff
@@ -9,7 +9,7 @@
  */
 public abstract class HealthDependency {
 
-  private Boolean primary = false;
+  private Boolean primary = true;
```

Comment:
Might be beneficial to add an alternative constructor that allows setting the primary value. An explicit default constructor would need to also be added for passivity.

This would allow implementors to just set the value in the constructor rather than overriding the `getPrimary` method.

Also, `getPrimary` isn't a great name. Could that method be deprecated and a new `isPrimary` method be added that returns the result of calling `getPrimary`? `isPrimary` could even be final to encourage using the new constructor instead of overriding `getPrimary`.

Reply:
All our other code that uses `getPrimary` can also be changed to call `isPrimary`

Reply:
Making it final won't affect consumers at all since it's new, and if it's implemented by just calling `getPrimary` it won't break consumers from a behavior perspective either.

Reply:
Oh, I see. I got `getPrimary` and `isPrimary` mixed up

Reply:
I made the new methods and created constructors here: https://github.com/cerner/beadledom/pull/123/commits/55cb09b555062c4ae1c016add5f38d260667fb36